config/src/main/resources/META-INF/nacos-db.sql

```diff
@@ -153,6 +154,7 @@ CREATE TABLE `his_config_info` (
   `src_ip` varchar(20) DEFAULT NULL,
   `op_type` char(10) DEFAULT NULL,
   `tenant_id` varchar(128) DEFAULT '' COMMENT '租户字段',
+  `secret_key` text NOT NULL COMMENT '秘钥',
```

Comment:
why not use name `encrypted_data_key`?

In your code, all relative named `encryptedDataKey` and `ENCRYPTED_DATA_KEY`, but in db is `secret_key`?

Reply:
encryptedDataKey is the original，But I think secret_key is more accurate

Reply:
> why not use name `encrypted_data_key`?
> 
> In your code, all relative named `encryptedDataKey` and `ENCRYPTED_DATA_KEY`, but in db is `secret_key`?

encryptedDataKey is the original，But I think secret_key is more accurate

Reply:
Let`s chat us  tomorrow